slam_vdb_ros/CMakeLists.txt

```diff
@@ -14,42 +15,39 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(sensor_msgs REQUIRED)
-
-add_subdirectory(third_party/nanovdb/nanovdb)
+find_package(Eigen3 REQUIRED)
+find_package(SLAM_VDB 1.0.0 PATHS ../slam_vdb/install)
```

Comment:
we should be able to use [Config mode](https://stackoverflow.com/a/20857070) instead of doing this.